lib/bundle/cask_dumper.rb

```diff
@@ -5,23 +5,45 @@ module CaskDumper
     module_function
 
     def reset!
+      @casks_data = nil
       @casks = nil
+      @casks_config = nil
+    end
+
+    def casks_data
```

Comment:
Could even call this `casks` and rename `casks` to `cask_names` or something below? Dunno how much effort renaming any callers will be so feel free to pass.